projects/packages/sync/src/class-listener.php

```diff
@@ -417,8 +417,16 @@ public function get_actor( $current_filter, $args ) {
 		);
 
 		if ( $this->should_send_user_data_with_actor( $current_filter ) ) {
-			require_once JETPACK__PLUGIN_DIR . 'modules/protect/shared-functions.php';
-			$actor['ip']         = jetpack_protect_get_ip();
+			$ip = false;
```

Comment:
As is this change will make it so that the ip of the actor is lost. If we don't want to move the shared-functions into a package at this time it would make sense to have a default value of:

$ip= isset( $_SERVER['REMOTE_ADDR'] ) ? $_SERVER['REMOTE_ADDR'] : '';

Which is what we do in the tracking package.

Reply:
Good point, updated the code.